### PR TITLE
Convert LinkMovementMethodExt to Kotlin

### PR DESCRIPTION
**Pull request by cooltey**

### CODE REVIEW

File: app/src/main/java/org/wikipedia/page/LinkMovementMethodExt.kt [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import android.text.Spannable
+import android.text.method.LinkMovementMethod
+import android.text.style.URLSpan
+import android.view.MotionEvent
+import android.widget.TextView
+import org.wikipedia.util.UriUtil
+import org.wikipedia.util.log.L
+
+class LinkMovementMethodExt : LinkMovementMethod {
+    fun interface UrlHandler {
```
Comment (sharvaniharan) [inline app/src/main/java/org/wikipedia/page/LinkMovementMethodExt.kt:12]: Empty line above this?

Comment (cooltey) [reply app/src/main/java/org/wikipedia/page/LinkMovementMethodExt.kt:13]: Done.

File: app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsImageTagsFragment.kt [hunk 2]
```diff
@@ -88,7 +88,7 @@ class SuggestedEditsImageTagsFragment : SuggestedEditsItemFragment(), CompoundBu
 
         binding.tagsLicenseText.text = StringUtil.fromHtml(getString(R.string.suggested_edits_cc0_notice,
                 getString(R.string.terms_of_use_url), getString(R.string.cc_0_url)))
-        binding.tagsLicenseText.movementMethod = LinkMovementMethodExt.getInstance()
```
Comment (dbrant) [inline app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsImageTagsFragment.kt:91]: What's the reason for this change? It doesn't seem related to Kotlin conversion.

**Review by dbrant — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsImageTagsFragment.kt (hunk 2 above)
Comment (cooltey) [reply app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsImageTagsFragment.kt:91]: Use `LinkMovementMethodExt.getInstance()` will show a "Unresolved reference: getInstance". 

Since this looks like a general usage of  `LinkMovementMethod` and it should be fine to use `LinkMovementMethod.getInstance()`.

**Review by sharvaniharan — APPROVED** [review]
